Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in

```diff
@@ -1002,14 +1002,17 @@
 (when (and (equal? (param "ENABLE_SANDBOX_MESSAGE_FILTER") "YES") (defined? 'mach-kernel-endpoint))
     (allow mach-kernel-endpoint
         (apply-message-filter
-            (allow mach-message-send (with report) (with telemetry))
+            (deny mach-message-send (with telemetry))
```

Comment:
Do we not want a report if we deny the message-send? Seems like if we are willing to pay the cost of telemetry, we should try to generate a report, too? Or do you think that will be a perf hit?